### PR TITLE
Podman stats with no containers is the same as podman stats --all

### PR DESCRIPTION
**Pull request by haircommander**
Fixes: #1028 

Signed-off-by: haircommander <pehunt@redhat.com>

### CODE REVIEW

**Comment by haircommander** [comment]
@mheon @rhatdan PTAL

File: cmd/podman/stats.go [hunk 1]
```diff
@@ -77,7 +77,9 @@ func statsCmd(c *cli.Context) error {
 
 	if ctr > 1 {
 		return errors.Errorf("--all, --latest and containers cannot be used together")
-	}
+	} else if ctr == 0 {
+		return errors.Errorf("--all, --latest, or a container must be specified")
+    }
```
Comment (mheon) [inline cmd/podman/stats.go]: Indentation here seems a little wonky - spaces instead of a tab?

**Comment by mheon** [comment]
One nit I'm pretty sure is going to trip gofmt, otherwise LGTM

File: cmd/podman/stats.go (hunk 1 above)
Comment (haircommander) [reply cmd/podman/stats.go]: forgot to fmt

**Comment by haircommander** [comment]
@mheon apparently stats should run with no containers, given the test that is failing. Is it worth changing the tests, or is this not really what we want functionality wise?

**Comment by wking** [comment]
> ... apparently stats should run with no containers, given the test that is failing.

Is there a `stats`-related failure?  It may just be flakes.  For example, [the smoketested][1] error:

```
setsebool:  SELinux is disabled.
neverallow check failed at /var/lib/selinux/targeted/tmp/modules/100/base/cil:9191
  (neverallow base_typeattr_7 unlabeled_t (file (entrypoint)))
    <root>
    allow at /var/lib/selinux/targeted/tmp/modules/200/container/cil:1554
      (allow spc_t unlabeled_t (file (entrypoint)))
    <root>
    allow at /var/lib/selinux/targeted/tmp/modules/100/sandboxX/cil:866
      (allow sandbox_x_domain exec_type (file (entrypoint)))
    <root>
    allow at /var/lib/selinux/targeted/tmp/modules/100/virt/cil:1677
      (allow virtd_lxc_t exec_type (file (entrypoint)))
    <root>
    allow at /var/lib/selinux/targeted/tmp/modules/100/virt/cil:2068
      (allow svirt_sandbox_domain exec_type (file (entrypoint)))

Failed to generate binary
/usr/sbin/semodule:  Failed!
```

[1]: https://s3.amazonaws.com/aos-ci/ghprb/projectatomic/libpod/80e11b8e583c62cb6dd2bb51925c2b516b7f68ae.1.1530306507759663393/output.log

**Comment by mheon** [comment]
Travis is going to fail, we broke it just prior to release today (one of the tests times out on travis nodes - oops. Got to increase the timeout there). The others look like there might be a legitimate problem. Let me test this.

**Comment by mheon** [comment]
Alright, Docker does allow stats with no container IDs, and seems to interpret it as `--all`... So we may be doing the right thing.

Still, we do have a bug here - we're not printing the header if there are no containers. Docker does do that.

**Comment by rhatdan** [comment]
@wking that is a bad libsemange package that we have worked around until it gets updated in fedora.  SHould be soon.  For now you should ignore.

**Comment by rhatdan** [comment]
@haircommander Ok lets have podman stats <NOARGS> imply --all

**Comment by mheon** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**